core_backend/app/utils.py

```diff
@@ -15,6 +16,15 @@
 )
 
 
+def generate_key() -> str:
+    """
+    Generate a new 32-character API key
+    """
+
+    # 32 bytes gives 43 characters here
+    return secrets.token_urlsafe(32)
```

Comment:
Should avoid magic numbers in code.

Don't need to put it in config but let's move it to the total as a global constant.
N_BYTES_SECRET_KEY or something similar.